client/src/js/services/grid/GridGrouping.js

```diff
@@ -21,6 +21,7 @@ function GridGroupingService(GridAggregators, uiGridGroupingConstants, Session,
   $timeout, util, uiGridConstants) {
   /** @const aggregators assigned by column ids */
   var DEFAULT_AGGREGATORS = GridAggregators.aggregators.tree;
+  var SELECTED_GROUP_HEADERS;
```

Comment:
The ALL_CAPITAL_LETTERS syntax is usually for constants.  Does this variable need to be global to the entire service?  It looks like it is only used in `hadnleBatchSelection()`.